_about/notes/0.8.md

```diff
@@ -8,4 +8,47 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model.
+[Learn more]({{home}}/docs/tasks/traffic-management-v1alpha3/)
+
+- **Envoy V2**. Pilot now uses the V2 Envoy API to program the proxies in the mesh, which increases
+effective scalability and eliminates spurious 404 errors.
+
+- **Constrain Inbound Ports**. Restrict the inbound ports in a pod to the ones needed by the apps running inside that pod.
+
```

Comment:
we should mention the istioctl converter tool to help convert the old traffic management yaml to the new one.